docs/index.md

```diff
@@ -81,12 +91,49 @@ to be named "my-commands" you'd put the following in your .bashrc or .zshrc:
 
 ```
 # posix example, e.g. .bashrc
-eval "`~/bin/tome init my-commands ~/my-scripts $0`"
+eval "$(tome init my-commands ~/my-scripts bash)"
+
+# zsh
```

Comment:
nitpick: should the "zsh" and "bash" be in their own code blocks as well? I see fish is the odd one out.

Reply:
Yes! I think I did that in the later PR.

Ok w/ leaving it as a nit that we address later in a docs cleanup task? If so, I'll add a placeholder issue to make a pass at documentation cleanup when most interface changes have landed.